packages/react-components/src/components/utils/filtrer-tree/sort.tsx

```diff
@@ -47,13 +47,26 @@ export const sortNodes = (nodes: TreeNode[], sortConfig: SortConfig[]): TreeNode
                 result = 0;
             } else {
                 if (typeof value1 === 'string' && typeof value2 === 'string') {
```

Comment:
I think value1 and value2 are strings by definition and already checked for null/undefined, so is this if-condition unnecessary?

Reply:
you're right ... value1 and value2 are strings.